### PR TITLE
include HumanTime

### PR DESCRIPTION
**Pull request by turnon**
handle mapping in module HumanTime, not in concrete classes

### CODE REVIEW

**Comment by allenan** [comment]
thanks for the PR, @turnon. much cleaner!

File: lib/human_time.rb [hunk 1]
```diff
@@ -4,33 +4,43 @@
...
+    def included base
+      add_alias base, greater_than_aliases, :>
+      add_alias base, greater_than_or_equal_to_aliases, :>=
+      add_alias base, less_than_aliases, :<
+      add_alias base, less_than_or_equal_to_aliases, :<=
+    end
+
+    private
+
+    def add_alias base, aliases, op
```
Comment (allenan) [inline lib/human_time.rb]: would you mind wrapping the arguments in parens?

File: lib/human_time.rb [hunk 2]
```diff
@@ -4,33 +4,43 @@
...
+      %w{older_than? comes_before? earlier_than?}
+    end
+
+    def less_than_or_equal_to_aliases
+      %w{older_than_or_equal_to? before_or_equal_to? earlier_than_or_equal_to?}
+    end
 
-  def self.less_than_or_equal_to_aliases
-    %w{older_than_or_equal_to? before_or_equal_to? earlier_than_or_equal_to?}
+    def included base
```
Comment (allenan) [inline lib/human_time.rb]: here as well :)

Comment (turnon) [reply lib/human_time.rb]: ok, I just pushed modification

**Comment by turnon** [comment]
ok, I just pushed modification

**Comment by allenan** [comment]
thanks! merging...

**Pull request merged**